source/_integrations/aussie_broadband.markdown

```diff
@@ -0,0 +1,45 @@
+---
+title: Aussie Broadband
+description: Instructions on how to integrate Aussie Broadband within Home Assistant.
+ha_category:
+- Network
+- Sensor
+ha_release: 2022.01
```

Comment:
```suggestion
ha_release: 2022.02
```
Sorry @nickw444 they skipped 01. You also apparently need to rebase this, if you give me write access to your branch again I can try do it.

Reply:
Please note, we don't use a leading zero.

`2022.2`